protocol/tests/keyed_ints.csv

```diff
@@ -0,0 +1,4 @@
+a,1,10
+b,2,20
+c,3,30
```

Comment:
Nit: you may need a newline at the end of the following line

Reply:
> Nit: you may need a newline at the end of the following line

Done